src/test/java/Homework16.java

```diff
@@ -0,0 +1,32 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.testng.Assert;
+
+import java.time.Duration;
+
+public class Homework16 extends BaseTest {
+  @test
```

Comment:
Hi,
Your homework 16 has been checked, but the test script not run because you use wrong TestNG Annotations.

TestNG Annotations are used to control the next method to be executed in the test script. TestNG annotations are defined before every method in the test code. In case any method is not prefixed with annotations, it will be ignored and not be executed as part of the test code. To define them, methods need to be simply annotated with ‘@Test‘.

//HINT: you need to add `import org.testng.annotations.Test;` and use `@Test` annotations not `@test`

Reply:
Hello,
I have corrected the work and I added the class in Gradle and was able to get it to run successfully. THe only problem I have now is that when I run it, koel will open and close quickly. Isn't koel supposed to stay open when you run test?